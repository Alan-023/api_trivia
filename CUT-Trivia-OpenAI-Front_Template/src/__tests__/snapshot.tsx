// This is just an example of how to use snapshot testing with Jest and React Testing Library
// Delete this file and __snapshots__ folder when you're ready to write your own tests

import { render } from '@testing-library/react';
import Home from '@/app/page';

it('renders homepage unchanged', () => {
  const { container } = render(<Home />);
  expect(container).toMatchSnapshot();
});
