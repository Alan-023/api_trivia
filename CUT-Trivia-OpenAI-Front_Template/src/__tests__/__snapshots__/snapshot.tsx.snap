// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div
    class="flex items-center justify-center border-2 h-full bg-slate-200"
  >
    <div
      class="flex flex-col text-center gap-5 items-center"
    >
      <img
        alt="Inbest Powered by Servnet"
        data-nimg="1"
        decoding="async"
        fetchpriority="high"
        height="25"
        src="/inbest.svg"
        style="color: transparent;"
        width="160"
      />
      <h1
        class="text-7xl font-bold"
      >
        Home Page
      </h1>
      <p
        class="text-3xl italic"
      >
        Everything starts here.
      </p>
      <div
        class="flex flex-col"
      >
        <a
          class="text-blue-500"
          href="/about"
        >
          Go to /about
        </a>
        <span
          class="text-xs text-slate-300"
        >
          This link is only for the cypress E2E test example
        </span>
      </div>
    </div>
  </div>
</div>
`;
